src/cmd/metric-store/app/metric_store.go

```diff
@@ -187,6 +187,9 @@ func (app *MetricStoreApp) startMetricsServer(tlsConfig *tls.Config) {
 		metrics.WithCounter(metrics.MetricStoreWrittenPointsTotal, prometheus.CounterOpts{
 			Help: "Number of points successfully written to storage engine",
 		}),
+		metrics.WithCounter(metrics.MetricStorePendingDeletionDroppedPointsTotal, prometheus.CounterOpts{
+			Help: "Number of points dropped due to deleting shard is in pending state",
+		}),
```

Comment:
Please replace with "Number of points dropped due to deleting shard which is in pending state"